cli/install/install.go

```diff
@@ -205,6 +211,27 @@ func getVersionsFromRepo(local bool, distfiles string, program string,
 	return versions, nil
 }
 
+// getCommit returns all available commits from repository.
+func getCommit(local bool, distfiles string, programName string,
```

Comment:
flag `isPullRequest` is unused here, can we simplify the code?

Reply:
fixed